Leanplum-SDK/Classes/Leanplum.m

```diff
@@ -2174,6 +2191,11 @@ + (void)forceContentUpdate
 }
 
 + (void)forceContentUpdate:(LeanplumVariablesChangedBlock)block
+{
+    [self forceContentUpdate:block includeContentAssignments:NO];
+}
+
++ (void)forceContentUpdate:(LeanplumVariablesChangedBlock)block includeContentAssignments:(BOOL)includeContentAssignments
```

Comment:
Same here. FCU with just another flag doesn't seem to be a good approach. We have a lot of flags that customer can enable.